cmd/admin-handlers.go

```diff
@@ -1199,15 +1200,52 @@ func (a adminAPIHandlers) OBDInfoHandler(w http.ResponseWriter, r *http.Request)
 		finish()
 	}
 
+	deadlineSeconds := time.Duration(3600)
+	deadline := r.URL.Query().Get("deadlineseconds")
```

Comment:
if we did stringification then we can simply use `time.ParseDuration(deadline)`

Reply:
that would work too

Reply:
Done